app/src/main/java/ch/epfl/favo/viewmodel/FavorViewModel.java

```diff
@@ -101,7 +105,17 @@ public void handleException(FirebaseFirestoreException e) {
   }
 
   @Override
-  public LiveData<Favor> getObservedFavor() {
+  public MutableLiveData<Favor> getObservedFavor() {

```

Comment:
Why did we change it to mutablelivedata? If the favor needs to be modified outside the view model, we should probably use MediatorLiveData with another setter method inside of the viewmodel https://developer.android.com/reference/android/arch/lifecycle/MediatorLiveData

Reply:
cuz I need to modify observed favor outside the view model.  will try the setter method in documentation.